imports/plugins/core/router/lib/router.js

```diff
@@ -313,15 +313,13 @@ Router.reload = () => {
  */
 Router.isActiveClassName = (routeName) => {
   const current = Router.current();
-  const group = current.route.group;
-  let prefix = "";
+  const { group } = current.route || {};
+  const defaultPrefix = "";
 
   if (current.route) {
-    const path = current.route.path;
-
-    if (group && group.prefix) {
-      prefix = current.route.group.prefix;
-    }
+    const { path } = current.route;
+    const { prefixArg } = group || {};
+    const prefix = prefixArg || defaultPrefix;
```

Comment:
Why define `defaultPrefix` at all if it's just used here? A comment that the empty string is the default would be fine.

Reply:
I think this could be `const { prefix } = group || { prefix: "" };` and could eliminate ln 321 and 317.

Reply:
Thanks for spotting. I think my attempt at the refactor was not so good. Updated it now.